content/en/releases/version-skew-policy.md

```diff
@@ -129,7 +129,11 @@ Pre-requisites:
 
 * The `kube-apiserver` instances these components communicate with are at **{{< skew currentVersion >}}** (in HA clusters in which these control plane components can communicate with any `kube-apiserver` instance in the cluster, all `kube-apiserver` instances must be upgraded before upgrading these components)
 
-Upgrade `kube-controller-manager`, `kube-scheduler`, and `cloud-controller-manager` to **{{< skew currentVersion >}}**
+Upgrade `kube-controller-manager`, `kube-scheduler`, and
+`cloud-controller-manager` to **{{< skew currentVersion >}}**. There is no
+required upgrade order between `kube-controller-manager`, `kube-scheduler`, and
+`cloud-controller-manager`. After you upgrade `kube-apiserver`, you can upgrade
```

Comment:
I would avoid "after" language relative to the API server if we can, since it's reasonable for all the control plane components to be upgraded simultaneously.

Would something like this work for the last sentence:

> You can upgrade these components in any order (or simultaneously), as long as they never speak to a `kube-apiserver` instance older than themselves.

Reply:
yeah, that sgtm